### ISSUE TITLE
Trouble running dgen_model.py

### ISSUE THREAD

**Issue by bmolina08** [NREL/dgen#26]
Have an error calculating system financial performance function
--------------
I was trying to run the model using the default example that you develop at the tutorial video (delaware case), I followed the video step by step but I am having an error in line 226 (solar_agents.chunk_on_row(financial_functions.calc_system_size_and_performance, sectors=scenario_settings.sectors, cores=cores, rate_switch_table=rate_switch_table). I have not changed anyting from the default case that you use as an example, so I do not know excatly why the error is coming out. It seems something is missing in the pandas dataframe that is been filling from the previous calculations, as the error says "single positional indexer is out-of-bounds"

### Details
I attached images of the error showed up at the spyder terminal
![Spyder terminal 1](https://user-images.githubusercontent.com/105063960/172661974-901f053d-2f51-4e22-ac15-fa41d810b40d.png)
![Spyder terminal 2](https://user-images.githubusercontent.com/105063960/172661996-2112fad0-d926-41a7-8707-e09be881b309.png)

**Comment by greg-otterpeak** [comment]
I'm getting the same error as well.   Is there any help out there?

**Comment by michaelavs** [comment]
Hi @greg-otterpeak,
I believe this issue arises due to which Pandas version you are using as some formatting changes have occurred for some functionality. Unfortunately, I am currently having some issues with the Conda environment on my side so I cannot fully replicate this issue and look into a solution. In the meantime, I would recommend trying to use Pandas version 1.1.5 and see if the issue resolves.

**Comment by greg-otterpeak** [comment]
Yeah. That was my first issue. I had to convert to the C version so it would work faster.

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by greg-otterpeak** [comment]
I have Python 3.11 installed, which seems to be a problem going back to the older version of Pandas.  

I have `pandas  2.1.4` installed.

```
conda install pandas=1.1.5
Retrieving notices: ...working... done
Channels:
 - defaults
Platform: win-64
Collecting package metadata (repodata.json): done
Solving environment: / warning  libmamba Added empty dependency for problem type SOLVER_RULE_UPDATE
failed

LibMambaUnsatisfiableError: Encountered problems while solving:
  - package pandas-1.1.5-py36hd77b12b_0 requires python >=3.6,<3.7.0a0, but none of the providers can be installed

Could not solve for environment specs
The following packages are incompatible
├─ pandas 1.1.5**  is installable with the potential options
│  ├─ pandas 1.1.5 would require
│  │  └─ python >=3.6,<3.7.0a0 , which can be installed;
│  ├─ pandas 1.1.5 would require
│  │  └─ python >=3.7,<3.8.0a0 , which can be installed;
│  ├─ pandas 1.1.5 would require
│  │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│  └─ pandas 1.1.5 would require
│     └─ python >=3.9,<3.10.0a0 , which can be installed;
└─ pin-1 is not installable because it requires
   └─ python 3.11.* , which conflicts with any installable versions previously reported.
```

**Comment by michaelavs** [comment]
I was able to get my Conda issues resolved and have not had this issue pop up. I am running the model with Pandas 2.1.4 and Python 3.10.13. I'm thinking this may be an issue with the load_profile_df not being read in or created correctly and instead returning an empty dataframe, hence Pandas saying that the first index is out of range. Can you add in something like `print("load_profile_df vals: ", load_profile_df)` 
above 
`pv['consumption_hourly'] = pd.Series(load_profile_df['consumption_hourly']).iloc[0]` in financial_functions.py and let me know what comes back from that? I would imagine an empty load_profile_df would have been caught much earlier on the model when load growth is applied to agents via `apply_load_growth` in elec.py, but sometimes bugs sneak through and are only caught when they compound later on.

**Comment by greg-otterpeak** [comment]
Good news!  I got it to work.  The problem was I forgot I downloaded the WA SQL and ran a DE scenario… ☹  Once I fixed that, it worked like a charm.

Thanks for clueing me in.
